pkg/controller/controller_binding.go

```diff
@@ -487,6 +531,9 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 				// over with a fresh view of the binding.
 				return err
 			}
+		} else if toUpdate.Status.OrphanMitigationInProgress {
```

Comment:
We only want to set `OperationStartTime` on the first time through for orphan mitigation. The way it is here, every reconciliation attempt will set the `OperationStartTime` anew, restarting the reconciliation retry window.